addons/web/static/src/js/views/basic/basic_model.js

```diff
@@ -636,6 +636,20 @@ var BasicModel = AbstractModel.extend({
             value: element.value,
             viewType: element.viewType,
         };
+
+        // If field type is selection, reorder values by registered selection order
+        // Used in Tree & Kanban views
+        // Task Id 2317536
```

Comment:
The taskid is already in the commit message. We don't need to add it in the comments.